runtime/src/lib.rs

```diff
@@ -1080,6 +1080,27 @@ impl membership_supernodes::Config for Runtime {
     type Event = Event;
 }
 
+parameter_types! {
```

Comment:
we're still using Substrate v3.0.0 (see Cargo.toml), not Substrate's latest 'master' branch.
so i'd suggest only adding the Asset pallet that we know is compatible with Substrate v3.0.0
https://github.com/paritytech/substrate/blob/v3.0.0/frame/assets/src/lib.rs#L1209
and list the constants in the same order.

Reply:
i just tried to git checkout v3.0.0 of Substrate, and the code actually appears the same as what you've done, so i think what you've already done should be sufficient

Reply:
Yeah. But we need to set right parameter for Asset pallet to make it work for our application.

Reply:
> Yeah. But we need to set right parameter for Asset pallet to make it work for our application.

yes, and bear in mind that we haven't done that for most of the existing parameters for other pallets yet.